DOCS-3510-TC-2.2-Add-Services/Gluster-screenshot (#1367)
truenas/documentation: content/TrueCommand/Clustering/ManageClusters.md

Check the **Confirm** box, then click **OK** to delete the cluster.

> On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select Services. Stop the Gluster service and uncheck START ON BOOT.

Reviewer: ```suggestion
On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.
```

Check the **Confirm** box, then click **OK** to delete the cluster.

> On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select Services. Stop the Gluster service and uncheck START ON BOOT.

Reviewer: ```suggestion
On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.
```

> On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.

Reviewer: Same question as on the other comment about gluster.
```suggestion
On the **Dashboard**, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.
```

Check the **Confirm** box, then click **OK** to delete the cluster.

> On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.

Reviewer: same comment as above. As this repeats, is this a good candidate for a snippet to maintain the content in one place?
```suggestion
On the **Dashboard**, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.
```

![DispersedClusterDelete](/images/TrueCommand/2.0/DispersedClusterDelete.png "DispersedClusterDelete")

Check the **Confirm** box, then click **OK** to delete the cluster.

> On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.

Reviewer: same as the above comments
```suggestion
On the **Dashboard**, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and clear the **START ON BOOT** checkbox.
```
